src/chain/block.cpp

```diff
@@ -138,43 +136,43 @@ bool block::operator!=(const block& other) const
 //-----------------------------------------------------------------------------
 
 // static
-block block::factory(const data_chunk& data, bool witness)
+block block::factory(const data_chunk& data, const bool witness)
```

Comment:
As a matter of style we never use `const` on non-ref parameters. Since these are always copies it does not affect the caller, but limits internals, which is not necessary.

Reply:
@evoskuil: Ok, I get it. I will remove it.

However, there are different views. For that reason, I would suggest adding your statement into the style guide. 
https://github.com/libbitcoin/libbitcoin-system/wiki/Style-Guide

Reply:
Will do, didn’t realize it wasn’t in there.

Reply:
I just updated the style guide and added a couple more missing items.